workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/request/client/HttpClient.java

```diff
@@ -18,6 +19,7 @@
 
 @Generated
 @Component
+@Slf4j
 public class HttpClient {
 
   public Response execute(String method, String url, Object body, Map<String, String> headers)
```

Comment:
as it is getting more complex maybe we should invest in wiremock or something to help us test this activity